docs/client/circe.md

```diff
@@ -12,10 +12,9 @@ libraryDependencies += "io.github.etspaceman" %% "kinesis4cats-client-logging-ci
 
 ```scala mdoc:compile-only
 import cats.effect._
-import software.amazon.awssdk.services.kinesis.KinesisAsyncClient
 
 import kinesis4cats.client.KinesisClient
 import kinesis4cats.client.logging.instances.circe._
 
-KinesisClient[IO](KinesisAsyncClient.builder().build(), kinesisClientCirceEncoders)
+KinesisClient.Builder.default[IO].withLogEncoders(kinesisClientCirceEncoders).build
```

Comment:
**suggestion:** maybe `KinesisClientBuilder`?

Reply:
I think I like the `.` here 😄